target/linux/ath79/dts/qca9531_comfast_cf-wr752ac.dts

```diff
@@ -0,0 +1,148 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qca953x.dtsi"
+
+/ {
+	compatible = "comfast,cf-wr752ac", "qca,qca9531";
+	model = "COMFAST CF-WR752AC";
+
+	aliases {
+		serial0 = &uart;
+		led-boot = &lan;
+		led-failsafe = &lan;
+		led-upgrade = &lan;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		pinctrl-names = "default";
+		pinctrl-0 = <&jtag_disable_pins &led_lan &led_wlan2g>;
+
+		lan: lan {
```

Comment:
If you insist on using this for led aliases, please prefix the _DTS label_ with "led_", so this becomes:
`led_lan: lan {`

Reply:
OK, if you don't mind that alias names get longer, I can do that.